Fix(Layout Shift Fix) - Resolved Layout Issues on DarkMode Toggle Button and Full-Screen Modal

nitro56565 opened the pull request:
<!--- Provide a formatted commit message describing this PR in the Title above -->
<!--- See our DEVELOPERS guide below: -->
<!--- https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format -->
<!--- Provide an overall summary of the pull request -->

## Summary
This PR addresses layout shift issues affecting the DarkToggle Button and Full-Screen Modal, ensuring a smoother and more consistent UI experience.

### Changes
<!--- More detailed and granular description of changes -->
<!--- These should likely be gathered from commit message summaries -->
- Fixed unintended layout shift on Full screen modal while on start guide tour.
- Adjusted positioning of the Darkmode button to prevent UI flickering and unnecessary clicks.

### Screenshots or Video
#### Before
![Screenshot From 2025-02-18 22-17-22](https://github.com/user-attachments/assets/58fbab62-d61d-4295-abfb-d34177b8c518)
![Screenshot From 2025-02-18 22-19-50](https://github.com/user-attachments/assets/2868b59a-dd41-406d-ab6a-dc93295f0cda)
#### After
![Screenshot From 2025-02-24 20-50-26](https://github.com/user-attachments/assets/8e59e5a2-e6b1-42f1-9443-ac25bfa4b0b4)
![Screenshot From 2025-02-24 20-51-13](https://github.com/user-attachments/assets/af0ccb11-c5db-47d7-8b9a-a9c5bc813c00)
![Screenshot From 2025-02-24 20-51-26](https://github.com/user-attachments/assets/8bab5ea3-99b6-4597-bbde-32ad8899c391)



### Author Checklist
- [X] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [ ] Vital features and changes captured in unit and/or integration tests
- [ ] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [ ] Extend the documentation, if necessary
- [ ] Merging to `main` from `fork:branchname`

DianaLease reviewed (commented):
I wonder if the tutorial should actually be targeting the "Preview Output" header in that step instead, rather than the full screen toggle button.

nitro56565 commented:
In the start tour, we generally cover the options, features, and usage of the playground. When David implemented the start tour, the full-screen modal was already a part of it. If I’m not mistaken, if you’re referring to including the entire header instead of just the full screen modal or dark mode toggle button, I can definitely work on clubbing it together.

nitro56565 commented:
@DianaLease I have updated the layout by combining the preview output with the two feature buttons: Dark Mode and Full-Screen Modal. Additionally, I have added a step in the start tour for the Dark Mode button, as it was missing before.

sanketshevkar commented:
Toggle button inside the preview block seems a bit off to me.

Scope of the toggle button is for the whole app and not just the preview block and the exapnd button's scope is just on the preview block. I don't feel they should be clubbed together.

nitro56565 commented:
so do you want to handle the toggle button to be outside the scope and the expand button inside the scope?

DianaLease commented:
@nitro56565 This looks good, although can you target the "Preview Output" header here instead of the expand button?
![Screenshot From 2025-02-24 20-51-13](https://github.com/user-attachments/assets/af0ccb11-c5db-47d7-8b9a-a9c5bc813c00)

nitro56565 commented:
@DianaLease Do you want me to target just the header, or would the whole preview window be a better choice?

sanketshevkar commented:
NIT: The dialog box is merging. Is it possible if we could undo that? Rest looks good.

Also you might to fix the failing test.

nitro56565 commented:
> NIT: The dialog box is merging. Is it possible if we could undo that? Rest looks good.
> 
> Also please fix the failing test.

sure @sanketshevkar I can handle the merging issue but from the above 3 options which one do you prefer for preview output, I believe the whole window would be a good option as referred below as well.


> ![Screenshot From 2025-02-25 22-59-50](https://github.com/user-attachments/assets/88e87db3-1bab-46f3-bda1-0087bb9e0f00)

DianaLease commented:
> sure @sanketshevkar I can handle the merging issue but from the above 3 options which one do you prefer for preview output, I believe the whole window would be a good option as referred below as well.
> 

as discussed on the call today, let's go with the whole preview window, thanks!
